src/connection-manager/auto-dialler.ts

```diff
@@ -113,7 +121,12 @@ export class AutoDialler implements Startable {
       return true
     })
 
-    // shuffle the peers
+    /**
+     * shuffle the peers
+     *
+     * todo: Be smarter about shuffling peers. We should add weights to peers
+     * that are more likely to succeed
```

Comment:
Not one for this PR but we may wish to sort peers by tag value so we try to connect to 'high value' peers first.  

Those that are KAD-close, those in GossipSub meshes, bootstrappers, etc.

Reply:
and those that we've successfully connected to in the past.